portfolio/src/main/webapp/index.html

```diff
@@ -41,9 +41,24 @@ <h3>More About Me</h3>
       <button onclick="addRandomQuote()">Quotes</button>
       <div id="quote-container"></div>
 
-      <p>Click here to see comments:</p>
+      <!--<p>Click here to see comments:</p>
       <button onclick="getComments()">Comments</button>
-      <div id="comment-container"></div>
+      <div id="comment-container"></div>-->
+
+      <form action="/data" method="POST">
+
+      <p>Comment Section:</p>
+      <textarea name="Comment-Input">Add a comment</textarea>
+      <br/><br/>
+
+      <p>Choose some options:</p>
+      <input type="Checkbox" name="upper-case" value="true"> Upper-case
```

Comment:
a checkbox allows the user to select multiple things -- what will happen if the user checks both options here?

Reply:
Yes. Needs to be either or, not both.

Reply:
i see that you fixed it by doing an either/or check in the code. for the future, you can also look at other form input types like <input type="radio"> because this one makes it so the user is only allowed to click one choice.